backend/bot/index.js

```diff
@@ -14,7 +29,23 @@ bot.on('start', async () => {
   }
 });
 
-const postResultToSlack = (match) => {
+const updateSlack = async (match) => {
+  // Update channel purpose
```

Comment:
upon separating the responsibilities through function extraction, you can drop this comment